Proyecto/src/main/java/com/example/Proyecto/controlador/Controlador.java

```diff
@@ -16,13 +15,14 @@ public String index() {
         return "index";
     }
 
+    // http://localhost:8080/procesa
     @PostMapping("/procesa")
-    public String procesa(HttpServletRequest request, Model model ) {
-        String nombres=" Axel David García Beltrán\n Camila Alexandra Cruz Miranda 316084707\n Fernando Gerardo Flores García\nIan Israel Garcia Vazquez";
+    public String procesa(HttpServletRequest request, Model model) {
+        String nombres = " Axel David García Beltrán\n Camila Alexandra Cruz Miranda\n Fernando Gerardo Flores García\n Ian Israel Garcia Vazquez";
         String nombre = request.getParameter("nombre");
-        System.out.println("nombre -> " + nombre+"\n"+nombres);
+        System.out.println("nombre -> " + nombre + "\n" + nombres);
         model.addAttribute("nombre", nombre);
-        return "nombre";
+        return "final";
```

Comment:
If you modify the previous one, it could keep as it is

Reply:
Ya lo dejamos con nombre, eliminamos el archivo final.html